src/modules/mc_pos_control/mc_pos_control_params.c

```diff
@@ -469,7 +481,31 @@ PARAM_DEFINE_FLOAT(MPC_ACC_UP_MAX, 5.0f);
  * @decimal 2
  * @group Multicopter Position Control
  */
-PARAM_DEFINE_FLOAT(MPC_ACC_DOWN_MAX, 5.0f);
+PARAM_DEFINE_FLOAT(MPC_ACC_DOWN_MAX, 2.0f);
```

Comment:
didn't you mention this was too low and that's why we increased it. the defaults should be resonable for upstream...